pages/login.js

```diff
@@ -0,0 +1,61 @@
+import Head from 'next/head';
+import { useState } from 'react';
+import styles from '../styles/Home.module.css';
+
+export default function Login() {
+  const [error, setError] = useState('');
+
+  function onSubmit(event) {
+    event.preventDefault()
+
+    // Get form data and create JSON object
+    const formData = new FormData(event.target)
+    const formDataObj = {}
+    formData.forEach((value, key) => (formDataObj[key] = value))
+
+    let username = formDataObj["username"]
+    let password = formDataObj["password"]
+
+    // Check if user input username and password
+    if (username.length == 0 || password.length == 0) {
+      setError("Please enter both a Username and Password.")
+      return
+    }
+    else {
+      setError("")
+    }
+
+    console.log(username)
+    console.log(password)
```

Comment:
Helps with logging readability

```suggestion
    console.log("Username: %s", username)
    console.log("Password: %s", password)
```